app/controllers/returns/AnotherBusinessExportWeightController.scala

```diff
@@ -69,7 +70,7 @@ class AnotherBusinessExportWeightController @Inject()(
 
         value =>
           request.userAnswers
-            .setOrFail(AnotherBusinessExportWeightPage, value)
+            .setOrFail(AnotherBusinessExportWeightPage, value, isCleanUp(mode))
```

Comment:
this maybe a rethinking or a better way to do it, but for the moment this is the quickest way to fix it